src/Microsoft.AspNetCore.Razor.Language/RazorParserFeatureFlags.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Razor.Language
+{
+    internal abstract class RazorParserFeatureFlags
+    {
+        internal static readonly RazorParserVersion LatestRazorParserVersion = RazorParserVersion.Version2_1;
+
+        public static RazorParserFeatureFlags Create(RazorParserVersion version)
+        {
+            if (version == LatestRazorParserVersion)
```

Comment:
This should be a >= `Version2_1` - we do don't want to keep changing this code if we add more versions